documentation/docs/04-compiler-and-api/04-custom-elements-api.md

```diff
@@ -73,6 +81,10 @@ When constructing a custom element, you can tailor several aspects by defining `
 ...
 ```
 
+These options don't exist in Svelte 3.
```

Comment:
I could go either way on this, but wondering if it should use `>`?
```suggestion
> These options don't exist in Svelte 3.
```

Reply:
Was going back and forth on this as well and thought it may be too prominent inside a box. But it's also better distinguishable/reading follow wise with the box.. mhm.

Reply:
Another way to do it might be to say "Starting in Svelte 4, " at the beginning of the first sentence of this section

Reply:
Yeah that might be better 👍